tjmonopix2/analysis/plotting.py

```diff
@@ -474,8 +474,13 @@ def create_cluster_size_plot(self):
     def create_cluster_tot_plot(self):
         ''' Create 1D cluster ToT plot '''
         try:
+            if np.max(np.nonzero(self.HistClusterTot)) < 128:
```

Comment:
This if clause makes sure that plot range of clustered ToT plot will always be at least the same (or larger) than the plot range of hit ToT. This will make comparison of the distributions easier in my opinion.